defi.tokenlist.json

```diff
@@ -643,6 +643,14 @@
             "symbol": "aAVAXb",
             "decimals": 18,
             "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/0x6C6f910A79639dcC94b4feEF59Ff507c2E843929/logo.png"
+        },
+        {
+            "chainId": 43114,
+            "address": "0xb00f1ad977a949a3ccc389ca1d1282a2946963b0",
```

Comment:
should be checksum cased

Reply:
Alright, thank you very much. I'll await your review.